.checkpatch.conf

```diff
@@ -8,5 +8,7 @@
 --ignore=BIT_MACRO
 --ignore=PREFER_PRINTF
 --ignore=PREFER_SCANF
+--ignore=VOLATILE
```

Comment:
To avoid these problems I've moved this patch series to the mailing list, so please refer to patch series http://patches.opendataplane.org/project/lng-odp/list/?series=240 for further reviews/comments on this request.